packages/core/src/database/index.ts

```diff
@@ -240,37 +240,38 @@ export const createDatabase = async ({
     const connection = (parse as unknown as typeof parse.parse)(
       preBuild.databaseConfig.poolConfig.connectionString!,
     );
+    if (!connection.database) {
```

Comment:
we reference connection.database further down in the code so it doesn't make sense to ignore it not existing.